star-rx72n-firmware/lib/rx_frame_ascii/src/rx_frame_ascii.c

```
@@ -0,0 +1,450 @@
+  return pos;
+}
+
+/**
+ * @brief Convert frame flags bitmask to human-readable string
+ * @param[in] flags Frame flags bitmask
+ * @param[out] buffer Output buffer for formatted string
+ * @param[in] buffer_len Size of output buffer in bytes
+ * @return Pointer to buffer (or empty string if buffer is NULL)
+ * @note Flags are formatted as pipe-separated values (e.g., "ACK|RETX|PRI")
+ */
+const char* rx_frame_ascii_flags_str(uint8_t flags, char* buffer, uint32_t buffer_len)
```

Clang (14.0.6)

[warning] pointer parameter 'buffer' can be pointer to const

(readability-non-const-parameter)